assets/scss/components/_button.scss

```diff
@@ -253,98 +228,102 @@ Class                   | Description
 
 .button {
 	@include buttonBase();
-	@include buttonHover();
-	@include color-all($C_textPrimary);
-	background: $C_contentBG;
-
-	.inverted &:hover {
-		background: $C_textSecondaryInverted;
-	}
-}
-
-.button--reset {
-	@include buttonHover(transparent);
-	background: transparent;
-	border-width: 0 !important;
-	border-radius: 0;
-	font-weight: normal;
-	min-height: 0;
-	padding: 0;
-
-	&[disabled] {
-		background: transparent !important;
-	}
+	@include buttonColor();
+	@include color-all($C_textSecondary);
 
 	.inverted & {
-		@include color-all($C_textPrimaryInverted);
+		@include buttonColor(
+			$bgColor: $C_fieldBGInverted,
+			$hoverColor: opacify($C_fieldBGInverted, .05),
+			$activeColor: opacify($C_fieldBGInverted, .1)
+		);
+		@include color-all($C_textSecondaryInverted);
```

Comment:
we could probably move these `color-all` mixins to the `buttonColor` mixin, using `getPrimaryTextColor` from the background color passed in: https://meetup.github.io/swarm-sasstools/sassdoc/index.html#function-getPrimaryTextColor